src/v/cluster/log_eviction_stm.cc

```diff
@@ -88,6 +88,11 @@ ss::future<> log_eviction_stm::write_raft_snapshots_in_background() {
           _delete_records_eviction_offset, _storage_eviction_offset);
         auto index_lb = _raft->log().index_lower_bound(evict_until);
         if (!index_lb) {
+            vlog(
+              _logger.debug,
```

Comment:
please use `_log` from `persisted_stm` this way we do not need to add ntp, it is already prefixed.